express-notes-api/index.js

```diff
@@ -0,0 +1,129 @@
+const express = require('express');
+const app = express();
+
+app.use('/api/notes', express.json());
+
+const fs = require('fs');
+
+const data = require('./data.json');
+
+app.get('/api/notes', (req, res) => {
+  const finalNotes = [];
+  for (const key in data.notes) {
+    finalNotes.push(data.notes[key]);
+  }
+  res.json(finalNotes);
+});
+
+app.get('/api/notes/:id', (req, res) => {
+  const notesNumber = req.params.id;
+  if (Number(notesNumber) <= 0 || !Number.isInteger(Number(notesNumber))) {
```

Comment:
Since you are using `Number` in multiple places it is best to just do that once and save it into a variable then use that variable for your checks:
```JAVASCRIPT
const notesNumber = Number(req.params.id);
```